packages/sdk-node/src/nodeTransport.ts

```diff
@@ -194,7 +194,7 @@ export class NodeTransport extends BaseTransport {
       }
     } else {
       try {
-        result = Buffer.from(result).toString('binary')
+        result = Buffer.from(result ?? '').toString('binary')
```

Comment:
Changed from `||` to `??` per @AprilArcus's comment to return empty string only when `null` or `undefined`.